contributors.txt

```diff
@@ -1,6 +1,11 @@
-﻿NV Access Limited
+﻿This is a list of people and organizations which contributed to the NVDA project in either way since the project has started. For an overview of code contributors, see also

```

Comment:
I don't believe 'either way' is a proper phrase here because it implies that it is possible to contribute only in two ways.